features/aragon/useAragon.ts

```diff
@@ -0,0 +1,52 @@
+import { contractHooksFactory } from '@lido-sdk/react';
+import { CHAINS } from 'config/chains';
+import { runWithFunctionLogger } from 'features/loggers';
+import { Aragon__factory } from 'generated';
+import { useCallback, useMemo } from 'react';
+import { useWeb3 } from 'reef-knot';
+
+const aragonAddressMap = {
+  [CHAINS.Mainnet]: '0x2e59A20f205bB85a89C53f1936454680651E618e',
+  [CHAINS.Goerli]: '0xbc0B67b4553f4CF52a913DE9A6eD0057E2E758Db',
+};
+
+export const useAragon = () => {
+  const { chainId } = useWeb3();
+
+  // TODO: get this from env
+  const aragonAddress =
+    chainId != null ? aragonAddressMap[chainId as CHAINS] : '';
```

Comment:
looks like you're not using our prettier config, 120 chars should fit this in single line

Reply:
Hm, it's actually not using shared config, will add one in other PR